lib/usecases/auth_token_storage_use_case.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter_survey/model/login_model.dart';
+import 'package:flutter_survey/usecases/base/base_use_case.dart';
+import 'package:flutter_survey/database/secure_storage.dart';
+import 'package:injectable/injectable.dart';
+
+@Injectable()
+class AuthTokenStorageUseCase extends DataSaverUseCase<LoginModel> {
+  final SecureStorage _secureStorage;
+
+  const AuthTokenStorageUseCase(
+    this._secureStorage,
+  );
+
+  @override
+  void save(LoginModel data) async {
+    await _secureStorage.writeSecureData(accessTokenKey, data.accessToken);
+    await _secureStorage.writeSecureData(refreshTokenKey, data.refreshToken);
+    await _secureStorage.writeSecureData(tokenTypeKey, data.tokenType);
+  }
```

Comment:
Do we have unit test for this new use case, please?

Reply:
I have added test in VM to check if the method `save` get invoked once. `writeSecureData` doesn't return any error/ exception so I didn't find any other way of testing it.

Reply:
What if we create a test case to verify that `_secureStorage.writeSecureData(x)` was called? 🙏

Reply:
Agree with @Tuubz.